src/notification/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { PureComponent, Fragment } from 'react'
+import { canUseDOM } from 'exenv'
+import cc from 'classcat'
+import { CrossIcon } from 'icon'
+import { createPortal } from 'react-dom'
+import { color, space } from '_utils/branding'
+import TransitionGroup from 'react-transition-group/TransitionGroup'
+import CustomTransition, { AnimationType } from 'transitions'
+import Button from 'button'
+import Text, { TextContainerType, TextDisplayType } from 'text'
+
+import style from './style'
+
+export interface NotificationProps {
+  readonly close: () => void
+  readonly isOpen: boolean
+  readonly className?: string
+}
+
+class Notification extends PureComponent<NotificationProps> {
+  private portalNode: HTMLElement
+  static defaultProps: Partial<NotificationProps> = {
+    isOpen: false,
+  }
+
+  constructor(props: NotificationProps) {
+    super(props)
+    if (canUseDOM) {
+      this.portalNode = document.createElement('div')
+      document.body.appendChild(this.portalNode)
+    }
+  }
+
+  componentWillUnmount() {
```

Comment:
Server side, this function isn't called, but it might be a good practice to "protect" every document code with `if (canUseDOM) {` ?

Reply:
done